test/fixtures/crd/web.yml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
`extensions/v1beta1` dropped in k8s `1.22`; replaced with `networking.k8s.io/v1`